.codecov.yml

```diff
@@ -0,0 +1,18 @@
+codecov:
+  notify:
+    require_ci_to_pass: yes
+
+coverage:
+  precision: 2
+  round: down
+  range: 50..100
+
+  status:
+    project:
+      default:
+        target: 90%
+        threshold: 1%
+    patch: no
+    changes: no
+
+comment: off
```

Comment:
I feel like we probably want the comment

Reply:
Ah, ok!